rcl/test/rcl/client_fixture.cpp

```diff
@@ -24,6 +24,33 @@
 #include "../memory_tools/memory_tools.hpp"
 #include "../scope_exit.hpp"
 #include "rcl/error_handling.h"
+#include "rcl/graph.h"
+
+bool
```

Comment:
Personally, I think `bool` is ok here, since it is a utility function just for the tests.